File: Lez_05_04_ProvaCSV/Lez_05_04_ProvaCSV/Classes/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lez_05_04_ProvaCSV.Classes
{
    internal class Persona
    {

        public string? Nome { get; set; }

        public string? Cognome { get; set; }
        public string? CodFis { get; set; }


        public Persona(string? nome, string? cognome, string? codFis)
        {
            Nome = nome;
            Cognome = cognome;
            CodFis = codFis;
        }

        public override string ToString()
        {
            return $"[PERSONA]{Nome} {Cognome} {CodFis}";
        }

        public string esportaCsv() 
        {
            return $"{Nome};{Cognome};{CodFis}";
        }



    }
}

File: Ts_Lezioni/ang_01_test/src/app/persona.ts
export class Persona {
  nome: string | undefined;
  cognome: string | undefined;
  eta: number | undefined;

  constructor(nome?: string, cognome?: string, eta?: number) {
    this.nome = nome;
    this.cognome = cognome;
    this.eta = eta;
  }
}
